caddyconfig/caddyfile/parse.go

```diff
@@ -368,7 +369,14 @@ func (p *parser) doSingleImport(importFile string) ([]Token, error) {
 		return nil, p.Errf("Could not import %s: is a directory", importFile)
 	}
 
-	importedTokens, err := allTokens(importFile, file)
+	input, err := ioutil.ReadAll(file)
+	if err != nil {
+		return nil, p.Errf("Could not read imported file %s: %v", importFile, err)
+	}
+
+	input = replaceEnvVars(input)
+
+	importedTokens, err := allTokens(importFile, bytes.NewReader(input))
```

Comment:
Maybe we should move replaceEnvVars into allTokens instead? We'd have to change the function signature from an io.Reader to []byte, but that's OK I think, it's unexported anyway.

Reply:
Ok, I might still read into an array - reworking the parsing logic to be stream based I think is a bit of work, it is friday here:)